### PR TITLE
added child attributes for operationsystem on show.json.rabl 

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/views/api/v1/operatingsystems/show.json.rabl [hunk 1]
```diff
@@ -1,2 +1,18 @@
 object @operatingsystem
 attributes :id, :name, :major, :minor, :family, :release_name
+
+child :media do
+    attributes :id, :name, :path, :media_path, :config_path, :image_path, :os_family
+end
+
+child :architectures do
+    attributes :id, :name
```
Comment (ohadlevy) [inline app/views/api/v1/operatingsystems/show.json.rabl]: cant we reuse the partial from the arch / show ?

**Comment by isratrade** [comment]
@ohadlevy Yes, I reused the partial for architectures and media.  Should I re-use the rabl partial for config_templates?  It has child node for operatingsystems and I don't want it to be self-referenced.

**Comment by ohadlevy** [comment]
@isratrade, DRY when it makes sense :)

**Comment by isratrade** [comment]
this can be closed.  Changes are in another pull request

**Pull request closed without merging**